wordpress-mobile/WordPress-iOS
# Issue/11587 Fix app blog URL

Fixes #11587. This PR just pulls over @DrBeta's changes from #11791 from their PR (thanks for picking up this issue!), merges in the latest `develop` and addresses the changes requested by @jkmassel.

**To test**

* Build and run
* Visit Me > App Settings > About WordPress for iOS
* Verify the blog address and that the site loads.

Update release notes:
